integreat_cms/cms/templates/pages/page_tree.html

```diff
@@ -180,6 +180,12 @@ <h1 class="heading">
                             {% endif %}
                         {% endwith %}
                     {% endif %}
+                    {% if request.user.is_staff %}
```

Comment:
```suggestion
                    {% if request.user.is_superuser or request.user.is_staff %}
```
In theory, the user model allows superusers to not be staff members, so for consistency, I'd also check superuser together with the staff flag everywhere it's used.